contrib/clojure-package/README.md

```diff
@@ -14,43 +14,41 @@ For a **video introduction**, see [Clojure MXNet with Carin Meier - Clojure Virt
 
 ## Current State and Plans
 
-The Clojure package is nearing the end of its first development milestone which is to achieve a close parity with the Scala package.
-
 Help is needed testing and generally making the package better. A list of the pacakge status and contribution needs can be found here [Clojure Package Contribution Needs](https://cwiki.apache.org/confluence/display/MXNET/Clojure+Package+Contribution+Needs). Please get involved :)
 
-Testing instructions can be found in the testing.md.
 
 ## Getting Started
 
+The Clojure MXNet framework consists of a core C library, a Scala Api that talks to it through JNI bindings, and finally a Clojure wrapper around the Scala api.
+
+Since there is a native code involved in the framework, what OS you are running matters.
+
 The following systems are supported:
 
 - OSX cpu
 - Linux cpu
 - Linux gpu
 
-There are two ways of getting going. The first way is the easiest and that is to use the pre-built jars from Maven. The second way is to build from source. In both cases, you will need to load the prereqs and dependencies, (like opencv).
-
-
-
-### Prerequisites
+There are three ways of getting started:
 
+* Use the Clojure jars with the native dependencies baked in. This the easiest way to get going.
+* Checkout the MXNet project from a release tag using the Scala jars with native deps. This is also a pretty easy way to get started.
+* Build from the MXNet project master. This option can be used to build the whole project yourself.
 
-Follow the instructions from https://mxnet.incubator.apache.org/install/osx_setup.html or https://mxnet.incubator.apache.org/install/ubuntu_setup.html
-about _Prepare Environment for GPU Installation_
-and _Install MXNet dependencies_
+### Getting Started with the Clojure Jars on Maven
 
+This option is one of the fastest. If you are looking to use MXNet in your own Clojure project, this is your best bet.
 
-#### Cloning the repo and running from source
+The Clojure MXNet jars are found on [maven.org](https://search.maven.org/search?q=clojure%20mxnet).
 
-To use the prebuilt jars (easiest), you will need to replace the native version of the line in the project dependencies with your configuration.
+- Create your own project with `lein new my-mxnet`
+- Edit your `project.clj` based on your system with the desired jar from maven.
 
-`[org.apache.mxnet/mxnet-full_2.11-linux-x86_64-gpu "1.3.0"]`
-or
-`[org.apache.mxnet/mxnet-full_2.11-linux-x86_64-cpu "1.3.0"]`
-or
-`[org.apache.mxnet/mxnet-full_2.11-osx-x86_64-cpu "1.3.0"]`
+   - clojure-mxnet-linux-gpu
+   - clojure-mxnet-linux-cpu
+   - clojure-mxnet-osx-cpu
```

Comment:
Listifying this is good. I find copy-pastable dependencies (like in the current README) helpful:

>`[org.apache.mxnet/mxnet-full_2.11-linux-x86_64-gpu "1.3.0"]` or `[org.apache.mxnet/mxnet-full_2.11-linux-x86_64-cpu "1.3.0"]` or `[org.apache.mxnet/mxnet-full_2.11-osx-x86_64-cpu "1.3.0"]`

Perhaps:

 - `[org.apache.mxnet.contrib.clojure/clojure-mxnet-linux-gpu "1.3.0"]` 
 - `[org.apache.mxnet.contrib.clojure/clojure-mxnet-linux-cpu "1.3.0"]`
 - `[org.apache.mxnet.contrib.clojure/clojure-mxnet-osx-cpu "1.3.0"]`

?

If the specific versions or Maven coordinates are unwanted due to change then I'd advocate for an explanatory sentence, perhaps with a link to https://search.maven.org/search?q=g:org.apache.mxnet.contrib.clojure